.circleci/run_tests.sh

```diff
@@ -134,6 +134,9 @@ sed -i 's/pool_mode = "transaction"/pool_mode = "session"/' .circleci/pgcat.toml
 # Reload config test
 kill -SIGHUP $(pgrep pgcat)
 
+# Revert settings after reload. Makes test runs idempotent
+sed -i 's/pool_mode = "session"/pool_mode = "transaction"/' .circleci/pgcat.toml
```

Comment:
The CircleCI container will be terminated in a moment, so this change won't be seen anywhere.